src/polyhedron.jl

```diff
@@ -61,6 +61,28 @@ function volume_simplex(s)
     return unscaled_volume_simplex(s) / factorial(fulldim(s))
 end
 
+"""
+    centerofmass(p::Polyhedron{T}) where {T}
+
+Returns the center of mass of `p`, represented as a `Vector{T}` of length `fulldim(p)`.
+Throws an error if `p` is degenerate.
+"""
+function centerofmass(p::Polyhedron{T}) where T
+    simplices = triangulation(p)
+    isempty(simplices) && error("Tried to compute center of mass of a degenerate polyhedron")
+    unscaled_center = zeros(T, fulldim(p))
+    unscaled_vol = zero(T)
```

Comment:
Add a comment explaining the approach here: e.g. the centroid is easy to compute (it's the average of the coordinates of the vertices and the centroid and center of mass coincides for simplices so we compute the center of mass of all simplicies to reduce the problem to the center of mass of finitely many objects for which we can just take the average weighted by the volume.

Reply:
Added.